src/tests/hci_verification.py

```diff
@@ -249,18 +249,18 @@ def hci_cin_bv_06_c(transport, idx, trace):
     trace.trace(6, "LE Clear White List Command returns status: 0x%02X" % status);
```

Comment:
Here and other places, "white list" should be replaced with "filter accept list"